cctp/src/handlers/messageTransmitter/MintInfo.ts

```diff
@@ -0,0 +1,54 @@
+import { IEventContext, IBind, Instance, ISecrets } from "@blockflow-labs/utils";
+import { mintTransactionsTable, ImintTransactionsTable } from "../../types/schema";
+import { attestationTable, IattestationTable } from "../../types/schema";
+
+/**
+* @dev Event::MessageReceived(address caller, uint32 sourceDomain, uint64 nonce, bytes32 sender, bytes messageBody)
+* @param context trigger object with contains {event: {caller ,sourceDomain ,nonce ,sender ,messageBody }, transaction, block, log}
+* @param bind init function for database wrapper methods
+*/
+export const MessageReceivedHandler = async (context: IEventContext, bind: IBind, secrets: ISecrets) => {
+// Implement your event handler logic for MessageReceived here
```

Comment:
The code is not readable, can you use a linter or prettier

Reply:
done sir